pkg/accounting/db.go

```diff
@@ -54,4 +83,6 @@ type DB interface {
 	QueryPaymentInfo(ctx context.Context, start time.Time, end time.Time) ([]*CSVRow, error)
 	// DeleteRawBefore deletes all raw tallies prior to some time
 	DeleteRawBefore(ctx context.Context, latestRollup time.Time) error
+	// ExceedsAlphaUsage checks if the usage limits are exceeded for a given project
+	ExceedsAlphaUsage(ctx context.Context, projectID uuid.UUID) (bool, error)
```

Comment:
`ExceedsAlphaUsage` is more of a domain logic thing, so probably should be part of the accounting package. The DB is more of methods `ProjectBandwidthUsage(ctx, project, from, to)` and `ProjectStorageUsage(ctx, project, from, to)`.